routes.js

```diff
@@ -4,6 +4,7 @@ const reviews = require('./controllers/review-controller');
 
 
 exports.route = (app) => {
+    app.get('/cocktail/:id', cocktail.getCocktailById)
```

Comment:
did you check to see if the getCocktailByName is still functional? 
essentially, /cocktail/:id and /cocktail/:name, is interpreted by the computer as /cocktail/:`information from the user`, which means it'll run both, one of, or neither functions.

Reply:
It's only running one at a time